lib/core/lib/core/models/opportunity/base.tsp

```diff
@@ -40,26 +39,28 @@ using CommonGrants.Types;
  * }
  * ```
  */
-@doc("A funding opportunity") // Overrides internal docstrings when emitting OpenAPI
+@doc("A funding opportunity")
 model OpportunityBase {
   /** Globally unique id for the opportunity */
   @visibility(Lifecycle.Read)
   id: uuid;
 
   /** Title or name of the funding opportunity */
+  @example("Small business grant program")
   title: string;
 
   /** Status of the opportunity */
   status: OppStatus;
 
   /** Description of the opportunity's purpose and scope */
+  @example("This program provides funding to small businesses to help them grow and create jobs")
   description: string;
 
   /** Details about the funding available */
-  funding: OppFunding;
+  funding?: OppFunding;
 
   /** Key dates for the opportunity, such as when the application opens and closes */
-  keyDates: OppTimeline;
+  keyDates?: OppTimeline;
```

Comment:
I made these optional since every field in the nested object is also optional. I can change that back if we think it should just be an empty object if these fields aren't set.